schema/gen/go/testUnions_test.go

```diff
@@ -107,40 +116,47 @@ func TestUnionKeyedComplexChildren(t *testing.T) {
 		}),
 	))
 
+	specs := []testcase{
+		{
+			name:     "InhabitantA",
+			typeJson: `{"String":"whee"}`,
+			reprJson: `{"a":"whee"}`,
+			typePoints: []testcasePoint{
+				{"", ipld.ReprKind_Map},
+				{"String", "whee"},
+				//{"SmolStruct", ipld.ErrNotExists{}}, // TODO: need better error typing from traversal package.
```

Comment:
nit: link to the issue in these TODOs? that way when the issue gets fixed, it's trivial to find the spots that can be improved